vendor.conf

```diff
@@ -1,57 +1,57 @@
 github.com/Shopify/logrus-bugsnag   6dbc35f2c30d1e37549f9673dd07912452ab28a5
-github.com/sirupsen/logrus          f006c2ac4710855cf0f916dd6b77acf6b048dc6e  # v1.0.3
+github.com/sirupsen/logrus          8bdbc7bcc01dcbb8ec23dc8a28e332258d25251f # v1.4.1
 github.com/bugsnag/bugsnag-go       13fd6b8acda029830ef9904df6b63be0a83369d0
-github.com/bugsnag/panicwrap	    e2c28503fcd0675329da73bf48b33404db873782
-github.com/bugsnag/osext	    0dd3f918b21bec95ace9dc86c7e70266cfc5c702
+github.com/bugsnag/panicwrap        e2c28503fcd0675329da73bf48b33404db873782
+github.com/bugsnag/osext            0dd3f918b21bec95ace9dc86c7e70266cfc5c702
 github.com/docker/distribution      83389a148052d74ac602f5f1d62f86ff2f3c4aa5
 github.com/opencontainers/go-digest a6d0ee40d4207ea02364bd3b9e8e77b9159ba1eb
-github.com/docker/go-connections    7395e3f8aa162843a74ed6d48e79627d9792ac55
+github.com/docker/go-connections    7395e3f8aa162843a74ed6d48e79627d9792ac55 # v0.4.0
 github.com/docker/go                d30aec9fd63c35133f8f79c3412ad91a3b08be06
 github.com/docker/go-metrics        399ea8c73916000c64c2c76e8da00ca82f8387ab
-github.com/dvsekhvalnov/jose2go     f21a8cedbbae609f623613ec8f81125c243212e6   # v1.3
-github.com/go-sql-driver/mysql      a0583e0143b1624142adab07e0e97fe106d99561   # v1.3
-github.com/gorilla/mux              53c1911da2b537f792e7cafcb446b05ffe33b996   # v1.6.1
+github.com/dvsekhvalnov/jose2go     f21a8cedbbae609f623613ec8f81125c243212e6 # v1.3
+github.com/go-sql-driver/mysql      a0583e0143b1624142adab07e0e97fe106d99561 # v1.3
+github.com/gorilla/mux              a7962380ca08b5a188038c69871b8d3fbdf31e89 # v1.7.0
```

Comment:
This one may need an update https://github.com/gorilla/mux/compare/v1.7.0...v1.7.2